src/css/flags/cast-available.less

```diff
@@ -1,10 +1,14 @@
 .jwplayer.jw-flag-cast-available {
+
     .jw-controlbar {
-      display: table;
+        display: table;
```

Comment:
Redundancy removed here: https://github.com/jwplayer/jwplayer/pull/1661/commits/b4ed0e538a6601ebc86d250e3696ab7308e259f3

Found we already accommodated for this in the idle state file.